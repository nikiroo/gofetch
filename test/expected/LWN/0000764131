            [$] LEARNING ABOUT GO INTERNALS AT GOPHERCON           

  [Front] Sep 5, 2018 19:20 UTC (Wed) (jake)

  o Reference: 0000764131
  o News link: https://lwn.net/Articles/764131
  o Source link: 


    [$]   Sorry,   this  article  is  currently  available  to  LWN
    suscribers only [https://lwn.net/subscribe/].



