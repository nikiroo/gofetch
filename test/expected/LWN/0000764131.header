0[$] Learning about Go internals at GopherCon	null/LWN/0000764131		70
i  [Front] Sep 5, 2018 19:20 UTC (Wed) (jake)
i
i    [1]GopherCon  is  the major conference for the [2]Go language ,
i    attended  by  1600  dedicated  "gophers", as the members of its
i    community  like  to  call  themselves.  Held  for the last five
i    years   in   Denver,   it   attracts  programmers,  open-source
i    contributors,  and  technical  managers  from  all  over  North
i    America  and the world. GopherCon's highly-technical program is
i    an  intense  mix  of  Go internals and programming tutorials, a
i    few of which we will explore in this article.
i    
i    
i    
i    [1] https://www.gophercon.com/
i    
i    [2]  https://golang.org/  Subscribers  can read on for a report
i    from GopherCon by guest author Josh Berkus.
i
