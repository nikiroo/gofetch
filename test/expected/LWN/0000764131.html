<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>[$] Learning about Go internals at GopherCon</h2>
	<div class='details'>([Front] Sep 5, 2018 19:20 UTC (Wed) (jake))</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0000764131</a></li>
        <li>News link: <a href='https://lwn.net/Articles/764131'>https://lwn.net/Articles/764131</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		[$] Sorry, this article is currently available to LWN suscribers only [https://lwn.net/subscribe/].
	</div>
<hr/>
</div>
</body>
